data/instrumentation.yaml

```diff
@@ -25,7 +25,7 @@ languages:
     urlName: go
     status:
       traces: stable
-      metrics: beta
+      metrics: stable
```

Comment:
```suggestion
      metrics: mixed (beta SDK & stable API)
```

Reply:
Hmm, I'm not sure how that will render. Let's commit the suggestion and see.

Reply:
<img width="696" alt="Screenshot 2023-07-25 at 16 55 29" src="https://github.com/open-telemetry/opentelemetry.io/assets/1615543/a35ffd2b-6c3e-442d-a18a-84799c16eff1">

I ran it locally here and it looks like this. Not sure if is going to be clear enough.